alphavilleseries/alphavilleseries_service_test.go

```diff
@@ -173,3 +174,28 @@ func cleanUp(assert *assert.Assertions, uuid string, seriesDriver service) {
 	assert.True(found, "Didn't manage to delete series for uuid %s", uuid)
 	assert.NoError(err, "Error deleting series for uuid %s", uuid)
 }
+
+func TestArraysEqual(t *testing.T) {
+	x := []string{"1", "2"}
+	y := []string{"2", "1"}
+
+	assert := assert.New(t)
+	assert.True(arraysEqual(x, y), "Arrays were equial but evaluated as not equal")
+}
+
+func arraysEqual(x []string, y[]string) bool {
```

Comment:
I don't think this is an appropriate name for the method. An array implicitly has an order, so ["1","2"] != ["2","1"]. This method is in fact checking for set-like equality (it's not checking for an equal count of elements in the case where a value is duplicated). Maybe `sameSetOfValues`?